### DISCUSSION TITLE
应该在首页加入翻译者是谁

### DISCUSSION THREAD

**Discussion by ghost** [luhaifeng666/obsidian-plugin-docs-zh#197] [General]
翻译官方文档的工作量之大，不管怎样都是值得肯定的。

**Comment by ghost** [comment]
非常感谢作者为社区所作的贡献！

**Reply by luhaifeng666** [reply]
感谢大佬支持~  🤩

**Comment by luhaifeng666** [comment]
> 翻译官方文档的工作量之大，不管怎样都是值得肯定的。

目前就我，所以加不加问题不大🤪
如果大佬有兴趣的话欢迎加入一起维护~
也许以后会有其他小伙伴会一起加入进来~

**Reply by ghost** [reply]
别这么说，我羞愧难当😂。我平时有空也可以写一些，但是我没有搭建这种 wiki 的相关经验，还不知道怎么操作。以及我一直很好奇的点是，搭建这种跟随官方一起更新的翻译文档要怎么知道哪些部分是最新的，哪些部分是过时的呢？

**Reply by luhaifeng666** [reply]
害，问题不大~
更新的话看官方文档的更新日志就好了，然后把变动的部分同步过来就好了。
我用的 Chrome 浏览器，装了个 Sourcegraph 插件，看差异比较清晰，就像这样：

<img width="1912" alt="image" src="https://user-images.githubusercontent.com/9375823/212663433-0227b495-bce9-405d-96b6-4ba1c6e04830.png">

**Reply by ghost** [reply]
昨晚研究了一下，VitePress 真是个好东西啊。我今天把 vue + naive-ui 写出来，不过我还从来没有 PR 过 😂 ，不是特别熟，见谅

**Reply by luhaifeng666** [reply]
> 昨晚研究了一下，VitePress 真是个好东西啊。我今天把 vue + naive-ui 写出来，不过我还从来没有 PR 过 😂 ，不是特别熟，见谅

好嘞！~ 辛苦辛苦~ 等你！[Smart]
